redis-commander fails to start up

I am on node v0.11.14 with npm v2.0.0. I installed redis-commander but when I try to run it I get
```
path.existsSync is now called `fs.existsSync`.
/Users/gauravarora/.nvm/v0.11.14/lib/node_modules/redis-commander/node_modules/browserify/index.js:204
        self[key] = w[key].bind(w);
                          ^
TypeError: Cannot read property 'bind' of undefined
  at /Users/gauravarora/.nvm/v0.11.14/lib/node_modules/redis-commander/node_modules/browserify/index.js:204:27
  at Array.forEach (native)
  at module.exports (/Users/gauravarora/.nvm/v0.11.14/lib/node_modules/redis-commander/node_modules/browserify/index.js:203:41)
  at module.exports (/Users/gauravarora/.nvm/v0.11.14/lib/node_modules/redis-commander/lib/app.js:48:11)
  at startWebApp (/Users/gauravarora/.nvm/v0.11.14/lib/node_modules/redis-commander/bin/redis-commander.js:158:3)
  at /Users/gauravarora/.nvm/v0.11.14/lib/node_modules/redis-commander/bin/redis-commander.js:116:10
  at /Users/gauravarora/.nvm/v0.11.14/lib/node_modules/redis-commander/lib/util.js:135:7
  at fs.js:295:14
  at Object.oncomplete (fs.js:93:15)
```

I now see:

$ redis-commander 
No config found or was invalid.
Using default configuration.
No Save: false
listening on  0.0.0.0 : 8081
Redis Connection 127.0.0.1:6379 Using Redis DB #0

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Cannot find module 'cmdparser' from '/Users/stillwell/scripts'
    at /usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:108:15)

$ sudo npm install -g redis-commander
/usr/local/bin/redis-commander -> /usr/local/lib/node_modules/redis-commander/bin/redis-commander.js
redis-commander@0.3.0 /usr/local/lib/node_modules/redis-commander
├── inflection@1.2.7
├── readline-browserify@0.0.4
├── cmdparser@0.0.3
├── sf@0.1.7
├── async@0.1.22
├── optimist@0.3.7 (wordwrap@0.0.2)
├── ejs@0.8.8
├── redis@0.7.3
├── node-redis-dump@0.2.1 (underscore@1.5.2, async@0.2.10, redis@0.8.6)
├── express@2.5.11 (mime@1.2.4, qs@0.4.2, mkdirp@0.3.0, connect@1.9.2)
├── browserify-middleware@3.0.1 (ms@0.6.2, mold-source-map@0.3.0, uglify-js@2.4.20, browserify@4.2.3)
└── browserify@9.0.8 (https-browserify@0.0.0, tty-browserify@0.0.0, builtins@0.0.7, constants-browserify@0.0.1, process@0.10.1, path-browserify@0.0.0, os-browserify@0.1.2, isarray@0.0.1, inherits@2.0.1, commondir@0.0.1, string_decoder@0.10.31, stream-browserify@1.0.0, defined@0.0.0, shell-quote@0.0.1, has@1.0.0, domain-browser@1.1.4, shallow-copy@0.0.1, xtend@3.0.0, duplexer2@0.0.2, deep-equal@1.0.0, querystring-es3@0.2.1, assert@1.3.0, timers-browserify@1.4.0, punycode@1.2.4, through2@1.1.1, util@0.10.3, events@1.0.2, vm-browserify@0.0.4, parents@1.0.1, read-only-stream@1.1.1, console-browserify@1.1.0, concat-stream@1.4.8, subarg@1.0.0, readable-stream@1.1.13, http-browserify@1.7.0, url@0.10.3, labeled-stream-splicer@1.0.2, shasum@1.0.1, buffer@3.2.0, browser-resolve@1.8.2, resolve@1.1.6, glob@4.5.3, JSONStream@0.10.0, deps-sort@1.3.5, browser-pack@4.0.1, browserify-zlib@0.1.4, insert-module-globals@6.2.1, syntax-error@1.1.2, crypto-browserify@3.9.14, module-deps@3.7.7)
stillwell:scripts stillwell$ redis-commander 
No config found or was invalid.
Using default configuration.
No Save: false
listening on  0.0.0.0 : 8081
Redis Connection 127.0.0.1:6379 Using Redis DB #0

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Cannot find module 'readline-browserify' from '/Users/stillwell/scripts'
    at /usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /usr/local/lib/node_modules/redis-commander/node_modules/browserify-middleware/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:108:15)
stillwell:scripts stillwell$

--- a/lib/app.js
+++ b/lib/app.js
@@ -9,6 +9,8 @@ var express = require('express');
 var browserify = require('browserify-middleware');
 var myUtils = require('./util');
 
+process.chdir( path.join(__dirname, '..') );    // fix the cwd
+
 var viewsPath = path.join(__dirname, '../web/views');
 var staticPath = path.join(__dirname, '../web/static');
 var redisConnections = [];

--- a/package.json
+++ b/package.json
@@ -39,5 +39,6 @@
   },
   "bin": {
     "redis-commander": "./bin/redis-commander.js"
-  }
+  },
+  "preferGlobal": true
 }

